src/generic/c1_plate_helper.cc

```diff
@@ -552,16 +552,16 @@ namespace C1PlateHelper
        oomph_info << std::endl;
       }
     }
-     
+
     // The final job is to constrain this duplication using the specialised
     // Lagrange multiplier elements which enforce equality of displacement and
     // its derivatives either side of this corner.
-    C1CurviLine* left_parametrisation_pt  =
-     c1_curviline_pt[boundary_on_which_node_is_left];
+    C1CurviLine* left_parametrisation_pt =
+      c1_curviline_pt.at(boundary_on_which_node_is_left);
```

Comment:
Have to use std::map.at() rather than [] as [] is non-const and we are passing a const ref now.

Reply:
Interesting (though, we do expect this entry to exist so there's no danger of accidentally filling up a "sparse matrix" like structure). Good catch, though, and I've never been forced to use this.